src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/DecompressionHandler.cs

```diff
@@ -36,20 +36,30 @@ public DecompressionHandler(DecompressionMethods decompressionMethods, HttpMessa
         internal bool GZipEnabled => (_decompressionMethods & DecompressionMethods.GZip) != 0;
         internal bool DeflateEnabled => (_decompressionMethods & DecompressionMethods.Deflate) != 0;
         internal bool BrotliEnabled => (_decompressionMethods & DecompressionMethods.Brotli) != 0;
+        
+        private bool EncodingExists(HttpHeaderValueCollection<StringWithQualityHeaderValue> acceptEncodingHeader, string encoding)
+        {
+            foreach (var existingEncoding in acceptEncodingHeader)
+            {
+                if (string.Equals(existingEncoding.Value, encoding, StringComparison.OrdinalIgnoreCase)) return true;
```

Comment:
`return true` should be on a separate line and enclosed by braces

Reply:
Done